Mesh_3/include/CGAL/Mesh_3/Triangulation_helpers.h

```diff
@@ -185,6 +185,9 @@ no_topological_change(Tr& tr,
                       const Point& p,
                       Cell_vector& cells_tos) const
 {
+  if(tr.point(v0) == p)
```

Comment:
```C++
typename Gt::Equal_3 equal = tr_.geom_traits().equal_3_object();
```
et caetera.

Reply:
Equal_3 doesn't exist for weighted points, see https://github.com/CGAL/cgal/issues/2639.

Reply:
But, what matter here is the equality of the 3D bare points, no?